src/components/sessions/activeSession/Question.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
```

Comment:
We should use a single component for display of single questions (RunningSession, QuestionPool..)